requirements.txt

```diff
@@ -0,0 +1,9 @@
+appdirs==1.4.3
+attrs==19.3.0
```

Comment:
Could you please specify why do you need all these dependencies?

Reply:
via 'black'